### ISSUE TITLE
New Feature => Copy IP to clipboard

### ISSUE THREAD

**Issue by christiangelone** [bovender/indicator-ip#14]
It would be a great idea to have a way to copy the ip to the clipboard, within the app

**Comment by bovender** [comment]
I agree that this would be useful. Unfortunately, I have switched to KDE a while ago and do not personally use this applet any more. But maybe I can set up a virtual machine with Ubuntu to add the feature. It may take some time though...

**Comment by Sispheor** [comment]
Duplicated with #9 .
Can be closed.
